.github/workflows/benchmark.yml

```diff
@@ -49,8 +49,6 @@ jobs:
           cache-dependency-path: js-framework-benchmark/package-lock.json
 
       - uses: Swatinem/rust-cache@v2
-        with:
-          working-directory: yew
```

Comment:
"Swatinem/rust-cache@v2" was updated and no longer understands this variable. Just gives a warning.